P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AutoMapper;
+using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void RoleService_CreateData()
+        {
+            // Arrange
+            Mock<IRoleRepository> roleRepository = new Mock<IRoleRepository>();
+            Mock<IMapper> mappingService = new Mock<IMapper>();
```

Comment:
Let's use `var` for these

Reply:
Also, I'd add "Mock" as a suffix for those variables, so we remember those are not the real objects.

Reply:
Done!